src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -207,13 +261,21 @@ private void LateUpdate()
         ClampCameraAltitude();
         ClampCameraXZPosition();
 
-        // It is mathematically incorrect to directly lerp on deltaTime like this, since we never get to the target (except by rounding I guess):
-        transform.position = Vector3.Lerp(transform.position, _targetPosition, Time.deltaTime * _panLerpSpeed);
-        transform.rotation = Quaternion.Slerp(transform.rotation, Quaternion.Euler(_rotateX, _rotateY, 0f), Time.deltaTime * _rotLerpSpeed);
+        // It is mathematically incorrect to directly lerp on
+        // deltaTime like this, since we never get to the target (except by rounding I guess):
```

Comment:
Does this need a TODO or FIXME tag?

Reply:
I don't think so, it was more of a comment to clarify what is happening in the code. I've tried to clarify it.